awx/main/models/credential/__init__.py

```diff
@@ -1039,11 +1039,11 @@ def create(self):
     },
     injectors={
         'env': {
-            'TOWER_HOST': '{{host}}',
-            'TOWER_USERNAME': '{{username}}',
-            'TOWER_PASSWORD': '{{password}}',
-            'TOWER_VERIFY_SSL': '{{verify_ssl}}',
-            'TOWER_OAUTH_TOKEN': '{{oauth_token}}',
+            'CONTROLLER_HOST': '{{host}}',
+            'CONTROLLER_USERNAME': '{{username}}',
+            'CONTROLLER_PASSWORD': '{{password}}',
+            'CONTROLLER_VERIFY_SSL': '{{verify_ssl}}',
+            'CONTROLLER_OAUTH_TOKEN': '{{oauth_token}}',
```

Comment:
Why not just repeat these with the `TOWER_*` names? Avoids any versioning problems. These are injectors, it does us no harm to put more stuff in.

Reply:
I have made the required changes.